docs/administration/deployment/upgrading-db/index.md

```diff
@@ -0,0 +1,87 @@
+cluster deployment from one REANA release series to another, such as
+from 0.7.* to 0.8.0, there might be changes in the REANA database schema
+between the two versions. In these cases, it is necessary to run a
+database upgrade script using [`alembic`](https://alembic.sqlalchemy.org/en/latest/).
+
+The `reana-db` command-line tool is provided to ease the database
+upgrade tasks. The tool is included in the `REANA-Server` component.
+After you upgrade REANA by running `helm upgrade` succcessfully, you can
+run the following command:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -- reana-db alembic upgrade
+```
+
+The output will show the upgrades accomplished.
+
+For instance, this is an example of the command and its output:
+
+```console
+$ kubectl default exec -i -t deployment/reana-server -- reana-db alembic upgrade
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade  -> c912d4f1e1cc, Quota tables.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
+```
+
+The `upgrade` command without any additional argument will attempt to
+upgrade the database to the latest revision. You can also pass a
+revision hash in the event that you just want to upgrade to a certain
+revision.
+
+Additionaly, you can check all the alembic `history` to list all the
+existing revisions:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -- reana-db alembic history
+f84e17bd6b18 -> 6568d7cb6710 (head), storing full workflow workspace.
+4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+ad93dae04483 -> 4801b98f6408, Job started and finished times.
+c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+<base> -> c912d4f1e1cc, Quota tables.
+```
+
+And also, get the exact hash of your `current` revision:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -- reana-db alembic current
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+6568d7cb6710 (head)
+```
+
+In some exceptional cases you might need to `downgrade` to a previous
+revision. For example, due to a failure during the release upgrade
+process, or because you were testing an alpha version and you want to
+revert to the previous state. To do this, you need to specify the exact
+hash of the revision you want to downgrade to. This example shows a
+command to downgrade the database schema to a previous state, right
+after the `Job started and finished times` were introduced.
```

Comment:
right after the database change "Job started and finished times" was introduced: